tests/cpp/tests/test_array.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include <sstream>
 
-#include "catch2/catch.hpp"
+#include "catch2/catch_all.hpp"
```

Comment:
Isn't the `catch_all.hpp` header significantly bigger than the original `catch2/catch.hpp` header? Is it slower to compile now?

Reply:
I see they recommend doing this replacement in the upgrade docs so I'm sure it's fine. Just curious.

Reply:
Ok. I changed to use more specific headers. Though the compilation time changed not at all so :shrug: .

Reply:
Disappointing!